cms/djangoapps/contentstore/views/course.py

```diff
@@ -727,6 +728,7 @@ def course_index(request, course_key):
             'mfe_proctored_exam_settings_url': get_proctored_exam_settings_url(course_module.id),
             'advance_settings_url': reverse_course_url('advanced_settings_handler', course_module.id),
             'proctoring_errors': proctoring_errors,
+            'pages_and_resources_mfe_link': get_pages_and_resources_url(course_key),
```

Comment:
You need to rebase your changes -- due to a recent change, this method now expects `course_key.id`

Reply:
@awaisdar001 I have pulled in the latest changes and the method expects a CourseLocator so this should work as course_key is a CourseLocator

Reply:
@awaisdar001 In your example course_item is not a CourseLocator object so you need the course_item.id to get the CourseLocator. But course_key is already the CourseLocator

Reply:
I have changed this to course_module.id now because the course_key is not available in every view and I wanted to keep the code consistent. Note: course_key == course_module.id